src/data/geo/point.js

```diff
@@ -0,0 +1,108 @@
+import Geometry from './geometry'
+import SpatialReferenceSystem from './spatial-reference-system'
+import { GeoTypes } from './constants'
+
+const GEOJSON_TYPE = GeoTypes.POINT
+const WKT_TYPE = GEOJSON_TYPE.toUpperCase()
+const PRECISION = .000000001
+
+const VALIDATION_MSG = 'In geospatial systems \'longitude\'(x) and \'latitude\'(y) should have correct values.'
+
+const validateX = (x, srs) => {
+  if ((srs === SpatialReferenceSystem.WGS84 || srs === SpatialReferenceSystem.PULKOVO_1995) && (x > 180 || x < -180)) {
+    throw new Error(VALIDATION_MSG)
+  }
+}
+
+const validateY = (y, srs) => {
+  if ((srs === SpatialReferenceSystem.WGS84 || srs === SpatialReferenceSystem.PULKOVO_1995) && (y > 90 || y < -90)) {
+    throw new Error(VALIDATION_MSG)
+  }
+}
+
+export default class Point extends Geometry {
+  constructor(srs) {
+    srs = srs || SpatialReferenceSystem.DEFAULT
+
+    super(srs)
+  }
+
+  validate() {
```

Comment:
in my previous comment I've mentioned about `validate` method, here it's a public and I assume it should be in each class

Reply:
but looking at `setX, setY` methods I can see that we need full validation only in the `constructor`, so maybe we should not specify `validate` method as public

Reply:
ok